File: documentation/user/en/query/filtering/examples/logical/not.java
final EvitaResponse<SealedEntity> entities = evita.queryCatalog(
	"evita",
	session -> {
		return session.querySealedEntity(
			query(
				collection("Product"),
				filterBy(
					not(
						entityPrimaryKeyInSet(110066, 106742, 110513)
					)
				)
			)
		);
	}
);

File: documentation/user/en/query/filtering/examples/logical/not.cs
EvitaResponse<ISealedEntity> entities = evita.QueryCatalog(
	"evita",
	session => session.QuerySealedEntity(
        Query(
        	Collection("Product"),
        	FilterBy(
        		Not(
        			EntityPrimaryKeyInSet(110066, 106742, 110513)
        		)
        	)
        )
	)
);